lesson08/assignment/vue-project/src/main.scss

```diff
@@ -0,0 +1,40 @@
+@import 'node_modules/bootstrap/scss/bootstrap';
+@import 'node_modules/bootstrap-vue/src/index.scss';
+
+body {
+  margin: 0;
+}
+
+// Jumbotron & Background Img 
+.jumbotron {
+  // margin-top: 3.5rem;
+  margin-bottom: 0 !important;
+  min-height: 100vh;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('https://images.unsplash.com/photo-1503726876301-67e54d319b60?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1443&q=80');
+  background-size: cover;
+  background-position: left;
+  background-repeat: no-repeat;
+
+  // Main heading 
+  h1 {
+    font-weight: 700;
+  }
+
+  // Alert Message
+  // .show-msg {
+  //   opacity: 0;
+  //   transition: opacity .5s ease-in;
+  //   -webkit-transition: opacity .5s ease-in;
+  //   -moz-transition: opacity .5s ease-in;
+  //   -o-transition: opacity .5s ease-in;
+  // }
+}
+
+// Tooltip bg color override
+.tooltip-inner {
+  background-color: $yellow !important;
+}
+
+.tooltip.bs-tooltip-right .arrow:before {
+  border-right-color: $yellow !important;
```

Comment:
`!important` is a tool of last resort. It is preferable to make a more specific selector that will override defaults. Would it be possible to add an id to these elements? Or nest them inside elements with ids? Then extremely specific selectors could be used.